npm/webpack-dev-server/src/startServer.ts

```diff
@@ -6,9 +6,14 @@ const debug = debugFn('cypress:webpack-dev-server:start')
 
 import { makeWebpackConfig } from './makeWebpackConfig'
 
-export async function start (userWebpackConfig = {}, { specs, config, devserverEvents }): Promise<WebpackDevServer> {
-  const foundConfig = fw.getWebpackOptions()
-  const webpackConfig = await makeWebpackConfig(foundConfig || userWebpackConfig, {
+export async function start (initialWebpackConfig, { specs, config, devserverEvents }): Promise<WebpackDevServer> {
+  if (initialWebpackConfig == null) {
+    debug('User did not pass in any webpack configuration')
+    // TODO: implement a method in find-webpack to return the path where it found the configuration
```

Comment:
I am not sure we should do this at all and provide any "default" webpack config. We will never be able to "match" users settings, so the custom config will be required. 

We probably should fail with error

Reply:
Yes I am inclined to agree... let's talk about this more as we get closer to our MVP and start polishing things for user testing. I think the best situation is we ask the user to provide a path to their webpack config. "magically" finding things never works.